### PR TITLE
 Fix #3068: F5 produces "Object reference not set to instance of object" error if REPL is not visible

### LINKED ISSUE #3068: F5 produces "Object reference not set to instance of object" error if REPL is not visible
subj - repros with any newly created project from R Project template.

### PR DESCRIPTION
**Pull request by int19h**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -76,7 +76,7 @@ src/Mocks/Editor/project.json
 src/Debugger/Impl/project.json
 src/R/Support/Impl/project.json
 src/R/Sql/Impl/project.json
-
+src/R/Wpf/Impl/project.json
```
Comment (AlexanderSher) [inline .gitignore]: Do we have project-dependent json in R.Wpf?

File: src/Package/Impl/ProjectSystem/RDebugLaunchProvider.cs [hunk 2]
```diff
@@ -36,6 +38,7 @@ namespace Microsoft.VisualStudio.R.Package.ProjectSystem {
     internal class RDebugLaunchProvider : DebugLaunchProviderBase {
         private readonly ProjectProperties _properties;
         private readonly IRInteractiveWorkflow _interactiveWorkflow;
+        private readonly IApplicationShell _shell;
```
Comment (AlexanderSher) [inline src/Package/Impl/ProjectSystem/RDebugLaunchProvider.cs]: Do you use this field?

File: .gitignore (hunk 1 above)
Comment (int19h) [inline .gitignore]: Yep. Apparently it was added some time ago, but project.json wasn't removed accordingly, so it kept popping up in the changelist and annoying me :)

File: src/Package/Impl/ProjectSystem/RDebugLaunchProvider.cs (hunk 2 above)
Comment (int19h) [inline src/Package/Impl/ProjectSystem/RDebugLaunchProvider.cs]: My bad, something that I forgot to remove from an older iteration.

**Comment by AlexanderSher** [comment]
👍

**Pull request merged**